modules/create-user-workloads-aws-edge.adoc

```diff
@@ -0,0 +1,136 @@
+
+.Procedure
+
+. Configure a `deployment` resource file named `<application_deployment>.yaml` for an application that you want to deploy to the edge compute node in the edge subnet.
++
+.Example `deployment` resource
+[source,yaml]
+----
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: <application_namespace> # <1>
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: <pvc_name> # <2>
+  namespace: <application_namespace>
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: gp2-csi # <3>
+  volumeMode: Filesystem
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: <application_name> # <4>
+  namespace: <application_namespace>
+spec:
+  selector:
+    matchLabels:
+      app: <application_name>
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: <application_name>
+        zone-group: ${ZONE_GROUP_NAME} # <5>
+    spec:
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
+      nodeSelector: # <6>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME}
+      tolerations: # <7>
+      - key: "node-role.kubernetes.io/outposts"
+        operator: "Equal"
+        value: ""
+        effect: "NoSchedule"
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: <application_name>
+          ports:
+            - containerPort: 8080
+          volumeMounts:
+            - mountPath: "/mnt/storage"
+              name: data
+      volumes:
+      - name: data
+        persistentVolumeClaim:
+          claimName: <pvc_name>
+----
+<1> Specify a namespace for your application.
+<2> #What do they need to do for the persistent volume claim?#
+<3> Specify the storage class name. For an edge compute configuration, you must use the `gp2-csi` storage class.
+<4> Specify a name for your application.
+<5> #How does this apply to Outposts?# Defines the group to where a zone belongs. For example, `us-east-1-iah-1`.
+<6> #How does this apply to Outposts?# Targets edge compute nodes that match the specified labels.
+<7> Specify tolerations that match the taints in the compute machine set for your edge compute machines. #The taint only has `key` and `effect`. What are `value` and `operator` doing here and how would the user know to set them?#
+
+. Create the `deployment` resource CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <application_deployment>.yaml
+----
+
+. Configure a `service` resource file named `<application_service>.yaml` that exposes a pod from a targeted edge compute node to services that run inside your edge network.
```

Comment:
We are declaring a NodePort service here and when creating Load Balancers. Do you thing we need to move it to LB section? Or say it may be optional depending of the service type they will be used: exposing with service LB, or through a custom ALB ingress.

Reply:
This makes me wonder if the user workloads topic belongs at the very end (after we go over ALB)